Combat/Data.cs

```diff
@@ -0,0 +1,82 @@
+﻿namespace BlueShadowMon
```

Comment:
Your created another Data.cs file! Data.cs is located in the root.
Move your changes to this file.